client/logs/bdqmpmboo3sunlgvprdg.log

```diff
@@ -0,0 +1 @@
+"hi yo hejho"
```

Comment:
I don't think these files supposed to be under version control. Missing a `.gitignore` directive?